input.py

```diff
@@ -0,0 +1,32 @@
+from typing import Counter
+from unittest.result import failfast
+from spellchecker import SpellChecker
+
+
+def function(sting):
+    userList = sting.split()
+    Counter=0
+    spell=SpellChecker()
+    misspelled=spell.unknown(userList)
+    for word in misspelled:
+        Counter = Counter +1
+    if(Counter!=0):
+        print ("Your input have error in it.... Exiting")
+        return None
+    else: 
+        
+        return userList
+
+def get_symptoms_from_user():
+    print("Enter the list of Symptoms by space: ")
+    i =input()
+    while (function(i) == None):

```

Comment:
Maybe the user wouldn't want to re-enter their symptoms. Should we watch for `exit`?

Reply:
Yeah that could work. Also, maybe the user has 3 attempts to get it right. That will be an easy fix.